### DISCUSSION TITLE
Package score in FAQ

### DISCUSSION THREAD

**Discussion by mattmassicotte** [SwiftPackageIndex/SwiftPackageIndex-Server#2993] [Ideas]
I was recently wondering if archived status affected a package’s score. I found the information! But it wasn’t as easy as I was expecting. I looked in the FAQ first. What do you think - should it be there?

**Comment by daveverwer** [comment]
Ah that FAQ is so out of date. 🫣 It needs a complete re-work really, but until I get around to that, adding something on the score would be a great idea. I'll convert this to an issue, thanks for reporting it @mattmassicotte!
